ted/src/main/java/frc/robot/control/Constants.java

```diff
@@ -104,6 +104,14 @@ public final class Constants {
     // velocity for fine placement
     public static final double FinePlacementRotationalVelocity = 0.7;
 
+    // ******************************************************************
+    //led constants
+    public static final int ledCanID = 30;
+    public static final int ledLength = 72;
+    public static final int ledStartIdx = 0;
+    public static final int ledBlinkDurationInHertz = 2; 
```

Comment:
Herz is a measure of frequency, not duration, so this should be ledBlinkFrequencyInHerz

Reply:
done